documentation/docs/reference/python.md

```diff
@@ -8,11 +8,11 @@ keywords:
 - type
 - reference
 ---
-# API Reference
+# Python API Reference
 
 :::info
 The following APIs will throw an exception if an error occurs.
-For all the optional values, the default values are the same as the ones in the [Rust API](../rust/api_reference.md).
+For all optional values, the default values are the same as in the [Rust API](rust.md).
 :::
 
```

Comment:
Not related to this change, but there is 
```
node = {
    'url': string,
    'auth': [NodeAuth](#nodeauth) (optional),
}
```
which should be 
```
node = {
    'url': string,
    'auth': [NodeAuth](#nodeauth) (optional),
    'disabled': bool,
}
```
could you update this in this PR?